subsys/bluetooth/host/audio/mcc.c

```diff
@@ -1758,6 +1824,37 @@ int bt_mcc_read_current_track_obj_id(struct bt_conn *conn)
 	return err;
 }
 
+int bt_mcc_set_current_track_obj_id(struct bt_conn *conn, uint64_t obj_id)
+{
+	int err;
+
+	if (!conn) {
+		return -ENOTCONN;
```

Comment:
If `conn == NULL`, then it doesn't really mean that is it not connected (that's a state in `conn`), but rather invalid argument. 
```suggestion
		return -EINVAL
```

Reply:
Fixed everywhere in mcc.c